lib/commons/aria/get-role.js

```diff
@@ -1,5 +1,83 @@
 import isValidRole from './is-valid-role';
 import getImplicitRole from './implicit-role';
+import lookupTable from './lookup-table';
+import { getNodeFromTree, closest } from '../../core/utils';
+import isFocusable from '../dom/is-focusable';
+import sanitize from '../text/sanitize';
+
+// when an element inherits the presentational role from a parent
+// is not defined in the spec, but through testing it seems to be
+// when a specific HTML parent relationship is required and that
+// parent has `role=presentation`, then the child inherits the
+// role (i.e. table, ul, dl)
+//
+// from Scott O'Hara:
+//
+// "the expectation for me, in standard html is that element
+// structures that require specific parent/child relationships,
+// if the parent is set to presentational that should set the
+// children to presentational.  ala, tables and lists."
+// "but outside of those specific constructs, i would not expect
+// role=presentation to do anything to child element roles"
+const inheritsPresentation = {
+	dd: ['dl'],
+	dt: ['dl'],
+	li: ['ul', 'ol'],
+	tbody: ['table'],
+	td: ['table'],
+	tfoot: ['table'],
+	th: ['table'],
+	thead: ['table'],
+	tr: ['table']
+};
+
+function hasGlobalAriaAttributes(vNode) {
+	return lookupTable.globalAttributes.find(
+		attr => vNode.hasAttr(attr) && sanitize(vNode.attr(attr))
+	);
```

Comment:
I tested this. The value doesn't matter. Even just doing `<button disabled role="none" aria-label>` triggers this behavior.

```suggestion
	return lookupTable.globalAttributes.find(attr => vNode.hasAttr(attr));
```